packages/web/src/components/MetaTags.tsx

```diff
@@ -78,22 +78,26 @@ export const MetaTags = (props: MetaTagsProps) => {
     author,
     children,
   } = props
-
+  const appTitle = global.__REDWOOD__APP_TITLE
```

Comment:
Please confirm if it's actually work, fail something in my tests. I don't know if it is just a bug with rwt on WSL.

Reply:
Maybe I can just write a test for that

Reply:
yeah this looks fine to me :)